lingua_franca/lang/parse_en.py

```diff
@@ -587,18 +586,43 @@ def extract_duration_en(text):
         'days': 0,
         'weeks': 0
     }
+    units = ['months', 'years', 'decades', 'centurys', 'millenniums'] + \
```

Comment:
these are spelled wrong on purpose because of the line below that strips the s

`unit_pattern = pattern.format(unit=unit_en[:-1])  # remove 's' from unit`